scripts/docker_run

```diff
@@ -17,6 +17,9 @@ docker_run_flags=(
   '--env=TERM=xterm-256color'
   '--env=BAZEL_REMOTE_CACHE_ENABLED'
   '--env=BAZEL_GOOGLE_CREDENTIALS'
+  '--env=OAK_KVM_TESTS'
+  '--env=RUST_BACKTRACE'
```

Comment:
I always find this notation confusing. What does this do? Should not it be '--env=RUST_BACKTRACE=1'?

Reply:
This just means that it passes whatever value the env variable is set to on the host, which is convenient because we can set it in a script or GHA file, without modifying this file every time